sdks/web-sdk/package.json

```diff
@@ -78,7 +78,6 @@
     "eslint-plugin-svelte3": "^4.0.0",
     "husky": "^8.0.1",
     "postcss": "^8.4.18",
-    "prettier": "^2.7.1",
```

Comment:
Im not sure about this one. I saw you moved it to the root pjson, but web-sdk shouldn't be dependent on the monorepo. it should always stay in a stage where we can move it anywhere else and its able to run standalone

Reply:
I can add it back but the way I see it as long as its part of the monorepo it will depend on it and once its not changes will be made then. To me monorepos do not share the same goal of microservices (100% self contained).